redbot/cogs/trivia/data/lists/hockey.yaml

```diff
@@ -0,0 +1,2097 @@
+- Toronto
+- Maple Leafs
+- Leafs
+What team hosted the 2003 NHL Entry Draft?:
+- Nashville Predators
+- Nashville
+- Predators
+What team hosted the 2004 NHL Entry Draft?:
+- Carolina Hurricanes
+- Carolina
+- Hurricanes
+What team hosted the 2005 NHL Entry Draft?:
+- Ottawa Senators
+- Ottawa
+- Senators
+What team hosted the 2006 NHL Entry Draft?:
+- Vancouver Canucks
+- Vancouver
+- Canucks
+What team hosted the 2007 NHL Entry Draft?:
+- Columbus Blue Jackets
+- Columbus
+- Blue Jackets
+What team hosted the 2008 NHL Entry Draft?:
+- Ottawa Senators
+- Ottawa
+- Senators
+What team hosted the 2009 NHL Entry Draft?:
+- Montreal Canadiens
+- Montreal
+- Canadiens
+What team hosted the 2010 NHL Entry Draft?:
+- Los Angeles Kings
+- LA Kings
+- LA
+- Kings
+- Los Angeles
+What team hosted the 2011 NHL Entry Draft?:
+- Minnesota Wild
+- Minnesota
+- Wild
+What team hosted the 2012 NHL Entry Draft?:
+- Pittsburgh Penguins
+- Pittsburgh
+- Penguins
+What team hosted the 2013 NHL Entry Draft?:
+- New Jersey Devils
+- New Jersey
+- Devils
+What team hosted the 2014 NHL Entry Draft?:
+- Philadelphia Flyers
+- Philadelphia
+- Flyers
+What team hosted the 2015 NHL Entry Draft?:
+- Florida Panthers
+- Florida
+- Panthers
+What team hosted the 2016 NHL Entry Draft?:
+- Buffalo Sabres
+- Buffalo
+- Sabres
+What team hosted the 2017 NHL Entry Draft?:
+- Chicago blackhawks
+- Chicago
+- Blackhawks
+What team hosted the 2018 NHL Entry Draft?:
+- Dallas Stars
+- Dallas
+- Stars
+What team hosted the 2019 NHL Entry Draft?:
+- Vancouver Canucks
+- Vancouver
+- Canucks
+Who was the first player in NHL history to score 100 points in a single season, when he scored 126 total points?:
+- Phil Esposito
+- Esposito
+Who had their Detroit Red Wing number 12 retired in 1995?:
+- Sid Abel
+- Abel
+Which player was the first in NHL to score a full-strength, power play, short-handed, penalty shot, and open net goal in the same game?:
+- Mario Lemieux
+- Lemieux
+Which NHL player has played in the most regular season wins during his career?:
+- Scott Stevens
+- Stevens
+Which player was the first to win more than two individual trophies in one season?:
+- Stan Mikita won the Art Ross trophy, Lady Bing trophy, and Hart trophy two years in a row with the Chicago Blackhawks in 1966-67 and 1967-68.
+- Stan Mikita
+- Mikita
+What undrafted goalie has the most games played?:
+- Ed Belfour (963)
+- Ed Belfour
+- Belfour
+What undrafted skater (non-goalie) has the most games played?:
+- Adam Oates (1337)
+- Adam Oates
+- Oates
+What undrafted skater (non-goalie) has the most goals?:
+- Dino Ciccarelli (608)
+- Dino Ciccarelli
+- Ciccarelli
+What undrafted skater (non-goalie) has the most points?:
+- Adam Oates (1337)
+- Adam Oates
+- Oates
+Which goaltender has the most games played?:
+- Martin Brodeur (1266)
+- Martin Brodeur
+- Brodeur
+Who is the shortest player to ever make the NHL?:
+- Roy Worters (5'3")
+- Roy Worters
+- Worters
+What country won the 2000 WJC?:
```

Comment:
This and plenty of following questions have abbreviations as correct answers. Not sure if want